docs/development.md

```diff
@@ -116,6 +116,21 @@ generated clients and other generated files.
 1.  Add the generated files to your PR, preferably in a separate, generated-only
     commit so that they are easier to review.
 
+## Verify Go source files
+
+To verify and fix your Go source files:
+
+1. Run `./hack/verify-go-src.sh -v --rootdir $(pwd)`
```

Comment:
Can you explain where you need to run this from to get `pwd` to work properly? I assume you have to be in the repo root.